### PR TITLE
chore(geo): Update pricing plan link for cli docs

### PR DESCRIPTION
**Pull request by Amplifiyer**
_Description of changes:_Update pricing plan link for cli docs

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/cli/geo/maps.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ Amazon Location Service offers three pricing plans namely, `RequestBasedUsage`,
 You select one of the following pricing plans for the Amazon Location Service resources you create for your application.
 Amplify helps you determine the best pricing plan for you by guiding you through a set of questions.
 These pricing plan related set of questions are presented to you when you add the first Geo resource to your application. 
-Once you chose the pricing plan, it will be automatically used for the subsequent Geo resources added to your application.
+Once you chose the pricing plan, it will be automatically used for the subsequent Geo resources added to your application. Review [Amazon Location Service Pricing](https://aws.amazon.com/location/pricing/) for more details.
```
Comment (phani-srikar) [inline docs/cli/geo/maps.md:61]: Link to pricing plan doc is already included ~ [line 95-96](https://github.com/aws-amplify/docs/pull/3506/files#diff-7f088c60496cbfa52d069e93a7cb8f559c97e4a159dcfc5a7c9bda704ffd8d1fR95).

**Review by phani-srikar — CHANGES_REQUESTED** [review]
Any changes to Maps w.r.t pricing plan, will have corresponding changes in search page too.

File: docs/cli/geo/maps.md (hunk 1 above)
Comment (Amplifiyer) [reply docs/cli/geo/maps.md:61]: That's interesting, @hdworld11 can you check what the requirements are then?

**Comment by phani-srikar** [comment]
This is addressed in https://github.com/aws-amplify/docs/pull/3575

**Pull request closed without merging**